### PR TITLE
Trunk 2809:Removing a description from an existing concept and trying to save should give a user friendly error message

### PR DESCRIPTION
**Pull request by SandeepRaparthi1**

### CODE REVIEW

**Comment by dkayiwa** [comment]
@SandeepRaparthi1 can you at least include the ticket summary in the pull request message, instead of just the ticket number?

**Comment by SandeepRaparthi1** [comment]
@dkayiwa Apologies for the Name thing Changed the name now :)

**Comment by SandeepRaparthi1** [comment]
@dkayiwa Made the required Changes :) please review it

File: webapp/src/main/webapp/WEB-INF/messages.properties [hunk 1]
```diff
@@ -504,6 +504,8 @@ Concept.find=Find Concept(s)
 Concept.retiredMessage=This concept is retired
 Concept.name=Concept Name
 Concept.name.atLeastOneRequired=At least one non-empty name is required
+Concept.Description.atLeastOneRequired=At least one non-empty Description is required
+Concept.Description.Existing.Required=Existing Description can't be Empty
```
Comment (djazayeri) [inline webapp/src/main/webapp/WEB-INF/messages.properties]: please fix capitalization of the message, e.g. D and E should be lowercase.

**Comment by dkayiwa** [comment]
@SandeepRaparthi1 remind me to reopen after you fix the merge conflicts.

**Pull request closed without merging**